lisp/forge-bitbucket.el

```diff
@@ -18,6 +18,10 @@
 ;; You should have received a copy of the GNU General Public License
 ;; along with Forge.  If not, see http://www.gnu.org/licenses.
 
+;;; Commentary:
+
+;;; Limited support for bitbucket.org in forge.
+
```

Comment:
Since the other libraries do not have such a header, don't add one here either.

Reply:
Fixed